packages/dataviews/src/item-actions.js

```diff
@@ -47,6 +47,32 @@ function DropdownMenuItemTrigger( { action, onClick } ) {
 	);
 }
 
+export function ActionModal( {
```

Comment:
The list view cannot use the ItemActions component, so this PR exports the two things it can actually use.